packages/react-adminer/src/components/Modal.tsx

```diff
@@ -0,0 +1,29 @@
+import { Modal as ModalAntd } from 'antd';
+
+interface Props {
+	title: string;
+	setIsModalVisible: React.Dispatch<React.SetStateAction<boolean>>;
+	isModalVisible: boolean;
```

Comment:
tip do budoucna: pojmenovani je fajn, ale tady je pouze modal takze by ti ideálně stačilo "isVisible", "setVisible" nemusíš tam cpát to slovo modal, když neočekáváš ani nic jiného.